Cargo.toml

```diff
@@ -17,6 +17,7 @@ lazy_static = "1.4"
 serde = { version = "^1.0", features = ["derive"] }
 serde_json = "^1.0"
 log = { version = "0.4", features = ["std"] }
+typetag = "0.1"
```

Comment:
Shall we put persistence behind a feature flag? It requires quite some dependencies that I can see might be problematic for some users.

Reply:
That's true, but I'm struggling to come up with useful examples for a memory-only use of OpenMLS.